website/source/docs/drivers/external/singularity.html.md

```diff
@@ -0,0 +1,197 @@
+    }
+    ```
+
+* [`binds`][bind] - (Optional) A user-bind path specification. This spec has the format `src[:dest[:opts]]`, where src and
+dest are outside and inside paths.  If dest is not given, it is set equal to src.
+Mount options ('opts') may be specified as 'ro' (read-only) or 'rw' (read/write, which
+is the default). Multiple bind paths can be given by a comma separated list.
+
+    ```hcl
+    config {
+      bind = [ "host/path:/container/path" ]
+    }
+    ```
+
+* [`overlay`][overlay] - (Optional) Singularity command action flag, to enable an overlayFS image for persistent data
+storage or as read-only layer of container.  Multiple overlay paths can be given by a comma separated list.
+
+    ```hcl
+    config {
+      overlay = [ "host/path/to/overlay" ]
+    }
+    ```
+
+* [`security`][security] - (Optional) Allows the root user to leverage security modules such as
+SELinux, AppArmor, and seccomp within your Singularity container.
+You can also change the UID and GID of the user within the container at runtime.
+
+    ```hcl
+    config {
+      security = [ "uid:1000 " ]
+    }
+    ```
+
+* `contain` - (Optional) Use minimal `/dev` and empty other directories (e.g. /tmp and $HOME) instead of sharing filesystems from your host.
+
+    ```hcl
+    config {
+      contain = "false"
+    }
+    ```
+
+* `workdir` - (Optional) Working directory to be used for `/tmp`, `/var/tmp` and $HOME (if -c/--contain was also used).
+
+    ```hcl
+    config {
+      workdir = "/path/to/folder"
+    }
+    ```
+
+* `pwd` - (Optional) Initial working directory for payload process inside the container.
+
+    ```hcl
+    config {
+      pwd = "/path/to/folder"
+    }
+    ```
+
+## Networking
+
+Currently the `Singularity` driver only supports host networking. For more detailed instructions on how to set up networking options, please refer to the `Singularity` user guides [singularity-network]
+
+## Client Requirements
+
+The `Singularity` driver requires the following:
+
+* 64-bit Linux host
+* The `linux_amd64` Nomad binary
+* The Singularity driver binary placed in the [plugin_dir][plugin_dir] directory.
+* [`Singularity`][Singularity] v3.1.1+ to be installed
+
+## Plugin Options<a id="plugin_options"></a>
+
+* `enabled` - The `Singularity` driver may be disabled on hosts by setting this option to `false` (defaults to `true`).
+
+* `Singularity_path` - The location in which all containers are stored (commonly defaults to `/var/lib/Singularity`). See [`Singularity-create`][Singularity-create] for more details.
+
+An example of using these plugin options with the new [plugin
+syntax][plugin] is shown below:
+
+```hcl
+plugin "nomad-driver-Singularity" {
+  config {
+    enabled = true
+    Singularity_path = "/var/lib/Singularity"
+  }
+}
+```
+Please note the plugin name should match whatever name you have specified for the external driver in the [plugin_dir][plugin_dir] directory.
+
+## Client Configuration
```

Comment:
I don't think this section is relevant, because folks will only be using this with 0.9.0 and later (unlike the `lxc` driver it was copied from, which had a different configuration setting for backwards compatibility).